### PR TITLE
chore(DNM): using latest cosmos proto builder for proto

### PR DESCRIPTION
**Pull request by gsk967**
## Description

closes: #XXXX

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -323,10 +322,9 @@ lint-fix:
 ##                                 Protobuf                                  ##
 ###############################################################################
 
-DOCKER_BUF := $(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace bufbuild/buf:1.8.0
-
-protoVer=v0.7
-protoImageName=tendermintdev/sdk-proto-gen:$(protoVer)
+protoVer=0.13.0
```
Comment (robert-zaremba) [inline Makefile:325]: I think this is not compatible with cosmos SDK v0.46

Comment (gsk967) [reply Makefile:325]: Ohh, we will wait for util v0.47 integration to our repo

Comment (robert-zaremba) [inline Makefile:325]: let's use v0.7 and then we can merge this PR

**Review by robert-zaremba — APPROVED** [review]
pre-approving. Please use `protoVer=v0.7`

File: Makefile (hunk 1 above)
Comment (gsk967) [reply Makefile:325]: there is no v0.7 for https://github.com/cosmos/cosmos-sdk/pkgs/container/proto-builder/versions?filters%5Bversion_type%5D=tagged

**Comment by gsk967** [comment]
@robert-zaremba  I am closing this pr , we will reopen this pr once we integrated cosmos v0.47

**Pull request closed without merging**